staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -68,27 +91,27 @@ func (o *AuditLogOptions) ApplyTo(c *server.Config) error {
 			}
 			c.AuditPolicyChecker = policy.NewChecker(p)
 		}
+		if o.WebhookFile != "" {
+			// TODO(ericchiang): How do we enable non-batch mode? Maybe a kubeconfig extension? Do we
+			// need to register a new type for the kubeconfig loader to parse?
+			batchMode := true
```

Comment:
How do we actually want to expose this? Through another flag? It looks like kubeconfig extensions (what I'd prefer to do) requires registering types.